src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -99,6 +101,7 @@
         ParametersDefinitionProperty.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "parameters"
         BooleanParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "booleanParam"
         StringParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "string"
+        ChoiceParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "choice" and "choiceParam"
```

Comment:
Well clearly if we have this new core baseline, `HAVE_SYMBOL == true`, so we can just delete all these switches.